app/stac_api/models.py

```diff
@@ -41,6 +41,35 @@
 # 'SRID=4326;POLYGON ((5.96 45.82, 5.96 47.81, 10.49 47.81, 10.49 45.82, 5.96 45.82))'
 BBOX_CH = str(_BBOX_CH)
 
+SEARCH_TEXT_HELP_ITEM = '''<div class=SearchUsage> Search Usage: <ul>
+                        <li> [arg] will make a non exact search checking if [arg] 
+                        is part of the Item path </li> <li> Multiple [arg] can be used,
+                        separated by spaces. This will search for all elements containing all
+                        arguments in their path </li><li> ["collectionID/itemID"] will make an
+                        exact search for the specified item. </li> </ul> Examples : <ul>
+                        <li> Searching for "pixelkarte" will return all items which have
+                        pixelkarte as a part of either their collection ID or their item ID
+                        </li> <li> Searching for "pixelkarte 2016 4" will return all items
+                        which have pixelkarte, 2016 AND 4 as part of their collection ID or
+                        item ID </li><li>Searching for
+                        "ch.swisstopo.pixelkarte.example/item2016-4-example" will yield only
+                        this item, if this item exists.</li></ul></div>'''
+
+SEARCH_TEXT_HELP_COLLECTION = '''<div class=SearchUsage> Search Usage: <ul> <li>
+                              [arg] will make a non exact search checking if [arg] is part of 
+                              the collection ID </li><li>Multiple [arg] can be used, separated 
+                              by spaces. This will search for all collections ID containing all 
+                              arguments.</li><li>["collectionID"] will make an exact search for
+                              the specified collection.</li></ul> Examples : <ul> <li>
+                              Searching for "pixelkarte" will return all collections which have
+                               pixelkarte as a part of their collection ID </li> <li>
+                              Searching for "pixelkarte 2016 4" will return all collection 
+                              which have pixelkarte, 2016 AND 4 as part of their collection ID
+                              </li><li> Searching for "ch.swisstopo.pixelkarte.example" will 
+                              yield only this collection, if this collection exists. Please 
+                              note that it would not return a collection named 
+                              ch.swisstopo.pixelkarte.example.2.</li> </ul></div>'''
+
```

Comment:
Because the multi-line seems to work here, I would suggest to do some styling here in order to make the code easier to read. When running locally it seems that the styling is not applied to this help text, like it is in the search hove ?